imap_processing/ialirt/l0/parse_mag.py

```diff
@@ -332,14 +333,21 @@ def process_packet(
         pkt_counter = grouped_data["pkt_counter"][
             (grouped_data["group"] == group).values
         ]
+        coarse_time = grouped_data["mag_acq_tm_coarse"][
+            (grouped_data["group"] == group).values
+        ]
 
         if not np.array_equal(pkt_counter, np.arange(4)):
-            logger.warning(
+            logger.info(
                 f"Group {group} does not contain all values from 0 to "
                 f"3 without duplicates."
             )
             continue
 
+        if (coarse_time == 0).all():
+            logger.info(f"Group {group} contains zero timestamps.")
```

Comment:
nitpick - I read this as "there are no timestamps in group" so maybe rephrase? 
```suggestion
            logger.info(f"Group {group} contains timestamps equal to zero.")
```